fields/evaluate.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2023 Searis AS
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fields
+
+import (
+	"encoding"
+	"encoding/json"
+	"fmt"
+)
+
+const (
+	DefaultAggregation AggregateMethod = iota
+	Count
+	Min
+	Max
+	Sum
+	Avg
+	StateHistogramSeconds
+	StateHistogramPercent
+	StateHistogramRate
```

Comment:
Add prefix `AggregationXxx`

Reply:
I think that's probably a good idea. I was worried about the length of `AggreageSateHistogramSeconds` and the rest of them. I suppose I could go for `AggregateStateSeconds` etc.

I'll change it.